examples/models/cpp/buildsystem-override/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.4.1)
+project(seldon_custom_model VERSION 0.0.1)
+
+set(CMAKE_CXX_STANDARD 14)
+
+find_package(seldon REQUIRED)
```

Comment:
What are the CMake rules to find the `seldon` library? Should we make those explicit or should we add instructions to add the `seldon` library to the path?

Reply:
This is on the cmake package in seldon, which is in the `incubation/wrappers/s2i/cpp` folder. Basically the typical flow would be installing the packages by running `make && make install`.